stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/plugin/factory/StreamOfflinePluginFactory.kt

```diff
@@ -54,13 +60,19 @@ public class StreamOfflinePluginFactory(
             recoveryEnabled()
         }.build()
 
-        val userStateFlow = MutableStateFlow<User?>(null)
-        chatClient.preSetUserListeners.add { user -> userStateFlow.value = user }
+        val scope = CoroutineScope(DispatcherProvider.IO)
+
+        val repos = RepositoryFacadeBuilder {
+            context(appContext)
+            scope(scope)
+            defaultConfig(io.getstream.chat.android.client.models.Config(connectEventsEnabled = true, muteEnabled = true))
+            ChatClient.instance().getCurrentUser()?.let(this::currentUser)
+            setOfflineEnabled(true)
```

Comment:
We should make this configurable by some config, but we can handle that in a separate task

Reply:
But does it make sense to disable offline support now that we are using the `OfflinePlugin`? The way to disable it now should be not setting the plugins.

Reply:
I think we could remove `setOfflineEnabled ` from RepositoryFacade, WDYT?

Reply:
As decided over chat, let's remove it in a future moment, but keep it for now.